tools/github/runDummyTests.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
What is the purpose of this?
Seems it's being used here.
https://github.com/apache/openwhisk/pull/5361/files#diff-fc5257f5200a46b6baab3ca657b694cc84dc77e3a5f9d71062b25fce994c02b4R70

Reply:
to test the build without having to run a full test... it is handy to debug secrets and so on